ProcessOptimizer/model_systems/color_pH.py

```diff
@@ -0,0 +1,339 @@
+
+    dLPrime = Lab2['L'] - Lab1['L']
+    dCPrime = c2Prime - c1Prime
+    dHPrime = 2.0 * math.sqrt(c1Prime * c2Prime) * \
+        math.sin((math.pi * (0.5 * dhPrime)) / 180.0)
+
+    sL = 1.0 + (0.015 * (lBarPrime - 50.0) * (lBarPrime - 50.0)) / \
+        math.sqrt(20.0 + (lBarPrime - 50.0) * (lBarPrime - 50.0))
+
+    sC = 1.0 + 0.045 * cBarPrime
+    sH = 1.0 + 0.015 * cBarPrime * t
+
+    dTheta = 30.0 * math.exp(-((hBarPrime - 275.0) / 25.0)
+                             * ((hBarPrime - 275.0) / 25.0))
+
+    cBarPrime7 = cBarPrime**7
+
+    rC = math.sqrt(cBarPrime7 / (cBarPrime7 + 6103515625.0))
+    rT = -2.0 * rC * math.sin((math.pi * (2.0 * dTheta)) / 180.0)
+
+    DE2000 = math.sqrt((dLPrime / (kL * sL)) * (dLPrime / (kL * sL)) + (dCPrime / (kC * sC)) * (dCPrime / (kC * sC)) +
+                       (dHPrime / (kH * sH)) * (dHPrime / (kH * sH)) + (dCPrime / (kC * sC)) * (dHPrime / (kH * sH)) * rT)
+
+    # If arbitury Keyword arg verbose=True then print out the below
+    if verbose == True:
+
+        decoration = '-'*20
+
+        print(decoration)
+        print('LAB Value Input')
+        print(decoration)
+        print(f"LAB1: {Lab1['L']}, {Lab1['a']}, {Lab1['b']}")
+        print(f"LAB2: {Lab2['L']}, {Lab2['a']}, {Lab2['b']}")
+        print(decoration)
+        print('Outputs')
+        print(decoration)
+        print(f'a1Prime: {round(a1Prime, 4)}')
+        print(f'a2Prime: {round(a2Prime, 4)}')
+        print(decoration)
+        print(f'cBar: {round(cBar, 4)}')
+        print(f'cBar7: {round(cBar7, 4)}')
+        print(decoration)
+        print(f'c1Prime: {round(c1Prime, 4)}')
+        print(f'c2Prime: {round(c2Prime, 4)}')
+        print(decoration)
+        print(f'h1Prime: {round(h1Prime, 4)}')
+        print(f'h2Prime: {round(h2Prime, 4)}')
+        print(decoration)
+        print(f'(abs)h1Prime - h2Prime: {round(abs(h1Prime - h2Prime), 4)}')
+        print(decoration)
+        print(f'hBarPrime: {round(hBarPrime, 4)}')
+        print(f'g: {round(g, 4)}')
+        print(f't: {round(t, 4)}')
+        print(f'sL: {round(sL, 4)}')
+        print(f'sC: {round(sC, 4)}')
+        print(f'sH: {round(sH, 4)}')
+        print(f'rT: {round(rT, 4)}')
+        print(decoration)
+        print(f'DE2000: {round(DE2000, 4)}')
+
+    if test == True:
+        a1Prime = round(a1Prime, 4)
+        a2Prime = round(a2Prime, 4)
+        c1Prime = round(c1Prime, 4)
+        c2Prime = round(c2Prime, 4)
+        h1Prime = round(h1Prime, 4)
+        h2Prime = round(h2Prime, 4)
+        hBarPrime = round(hBarPrime, 4)
+        g = round(g, 4)
+        t = round(t, 4)
+        sL = round(sL, 4)
+        sC = round(sC, 4)
+        sH = round(sH, 4)
+        rT = round(rT, 4)
+        DE2000 = round(DE2000, 4)
+
+        return a1Prime, a2Prime, c1Prime, c2Prime, h1Prime, h2Prime, hBarPrime, g, t, sL, sC, sH, rT, DE2000
+
+    else:
+        return DE2000
+
+
+def find_closest_match(file_name, target_vector, target_columns):
```

Comment:
```suggestion
def find_closest_match(file_name, target_vector, target_1, target_2):
```
You are assuming there is exactly two targets. If that is what we do, I would prefer having two inputs.